packages/toolpad-app/src/toolpadDataSources/local/server.ts

```diff
@@ -101,8 +106,8 @@ async function createMain(): Promise<string> {
     async function getResolvers() {
       if (!resolversPromise) {
         resolversPromise = (async () => {
-          const functions = await import(${JSON.stringify(
-            relativeFunctionsFilePath,
+          const functions = await import(${pathToNodeImportSpecifier(
```

Comment:
doesn't this have to be the other way around? json.stringify on the outside

Reply:
Maybe, it's broken on Mac now, I'm fixing it.

Reply:
json.stringify is there to add quotes around the string and correctly escape its content

Reply:
Yeah that was just it, I was just having some weird issues on windows VSCode.